themes/SuiteP/tpls/Home.tpl

```diff
@@ -70,16 +70,16 @@
 
 
 <!-- CSS Files -->
-<link type="text/css" href="{sugar_getjspath file='custom/include/SugarCharts/js/Jit/Examples/css/base.css'}"
+<link type="text/css" href="{sugar_getjspath file='include/SugarCharts/js/Jit/Examples/css/base.css'}"
```

Comment:
still incorrect should be include/SugarCharts/Jit/css/base.css?, but is probably legacy and should be removed as with below